packages/gafl-webapp-service/src/pages/summary/licence-summary/__tests__/licence-summary.spec.js

```diff
@@ -107,33 +117,41 @@ describe('The licence summary page', () => {
   })
 
   it('licence length amendment causes a redirect to the summary page', async () => {
+    await injectWithCookies('GET', LICENCE_SUMMARY.uri)
+    const data = await injectWithCookies('GET', LICENCE_LENGTH.uri)
+    // Tests that the back-link is now set to the summary page
+    expect(data.payload.search(backLinkRegEx(LICENCE_SUMMARY.uri)) > 0).toBeTruthy()
```

Comment:
Should this be a separate test (that the back link updates), to keep tests simple?

Reply:
done